roles/cnf_cert/tasks/tests.yml

```diff
@@ -66,10 +66,10 @@
 
 # For v5.1.1 in advance, keep using some environment variables to
 # keep the definition of tnf_env_vars, but would be deprecated at some point.
-- name: Common tasks for CNF Cert Suite execution (tnf >= v5.1.1 and HEAD)
-  when: test_network_function_version is version("v5.1.1", ">=") or test_network_function_version == "HEAD"
+- name: Common tasks for CNF Cert Suite execution (tnf >= v5.1.3 and HEAD)
```

Comment:
This cannot be changed, else the flow will be broken for tnf v5.1.1 and v5.1.2.

What we only need to change is what I said in my prev comment:

_So, we need to update these two vars: https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/main/roles/cnf_cert/tasks/tests.yml#L76-L77_

_Where:_ `(test_network_function_version == 'HEAD')` _must be now_ `(test_network_function_version is version("v5.1.3", ">=") or test_network_function_version == "HEAD")`

So, we only need to modify these two vars, that's all

Reply:
my bad, I think I got this time : (